server/conf_test.go

```diff
@@ -308,6 +308,7 @@ func TestParseWrongTypes(t *testing.T) {
 	expectFailureFor(t, "store_limits:{channels:{\"foo.*bar\":{}}}", wrongSubjErr)
 	expectFailureFor(t, "store_limits:{channels:{\"foo.>.>\":{}}}", wrongSubjErr)
 	expectFailureFor(t, "store_limits:{channels:{\"foo..bar\":{}}}", wrongSubjErr)
+	expectFailureFor(t, "store_limits:{channels:{\"foo/bar\":{}}}", wrongSubjErr)
```

Comment:
For consistency with other changes, should these `wrongSubjErr`s be changed to `wrongChanErr`?